CRITICAL - 2021-02-01 00:58:31 --> Column 'username' cannot be null
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `fe...')
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fe...')
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fe...')
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fe...', Array, false)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert(Array, false)
#6 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\Home.php(99): CodeIgniter\Model->save(Array)
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->contact()
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#12 {main}
CRITICAL - 2021-02-01 00:58:42 --> Column 'username' cannot be null
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `fe...')
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fe...')
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fe...')
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fe...', Array, false)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert(Array, false)
#6 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\Home.php(99): CodeIgniter\Model->save(Array)
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->contact()
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#12 {main}
CRITICAL - 2021-02-01 01:01:31 --> Column 'username' cannot be null
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `fe...')
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fe...')
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fe...')
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fe...', Array, false)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert(Array, false)
#6 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\Home.php(99): CodeIgniter\Model->save(Array)
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->contact()
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#12 {main}
CRITICAL - 2021-02-01 01:01:38 --> Column 'username' cannot be null
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `fe...')
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fe...')
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fe...')
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fe...', Array, false)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert(Array, false)
#6 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\Home.php(99): CodeIgniter\Model->save(Array)
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->contact()
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#12 {main}
CRITICAL - 2021-02-01 01:03:28 --> Column 'username' cannot be null
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `fe...')
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fe...')
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fe...')
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fe...', Array, false)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert(Array, false)
#6 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\Home.php(99): CodeIgniter\Model->save(Array)
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->contact()
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#12 {main}
CRITICAL - 2021-02-01 01:05:15 --> Column 'username' cannot be null
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `fe...')
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fe...')
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fe...')
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fe...', Array, false)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert(Array, false)
#6 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\Home.php(99): CodeIgniter\Model->save(Array)
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->contact()
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#12 {main}
CRITICAL - 2021-02-01 01:05:49 --> Column 'username' cannot be null
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `fe...')
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fe...')
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fe...')
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fe...', Array, false)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert(Array, false)
#6 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\Home.php(99): CodeIgniter\Model->save(Array)
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->contact()
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#12 {main}
CRITICAL - 2021-02-01 01:09:45 --> Duplicate entry '0' for key 'PRIMARY'
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `fe...')
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fe...')
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fe...')
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fe...', Array, false)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert(Array, false)
#6 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\Home.php(99): CodeIgniter\Model->save(Array)
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->contact()
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#12 {main}
CRITICAL - 2021-02-01 21:33:56 --> Resolving timed out after 10000 milliseconds
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\FacebookClient.php(216): Facebook\HttpClients\FacebookCurlHttpClient->send('https://graph.f...', 'GET', '', Array, 60)
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(277): Facebook\FacebookClient->sendRequest(Object(Facebook\FacebookRequest))
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(226): Facebook\Authentication\OAuth2Client->sendRequestWithClientParams('/oauth/access_t...', Array)
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(166): Facebook\Authentication\OAuth2Client->requestAnAccessToken(Array)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Helpers\FacebookRedirectLoginHelper.php(228): Facebook\Authentication\OAuth2Client->getAccessTokenFromCode('AQBkrM1db7_6cCE...', 'http://localhos...')
#5 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\FB.php(41): Facebook\Helpers\FacebookRedirectLoginHelper->getAccessToken()
#6 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\FB->callback()
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FB))
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#11 {main}
CRITICAL - 2021-02-01 21:34:58 --> Resolving timed out after 10000 milliseconds
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\FacebookClient.php(216): Facebook\HttpClients\FacebookCurlHttpClient->send('https://graph.f...', 'GET', '', Array, 60)
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(277): Facebook\FacebookClient->sendRequest(Object(Facebook\FacebookRequest))
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(226): Facebook\Authentication\OAuth2Client->sendRequestWithClientParams('/oauth/access_t...', Array)
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(166): Facebook\Authentication\OAuth2Client->requestAnAccessToken(Array)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Helpers\FacebookRedirectLoginHelper.php(228): Facebook\Authentication\OAuth2Client->getAccessTokenFromCode('AQD4ZHDOEwP0tBI...', 'http://localhos...')
#5 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\FB.php(41): Facebook\Helpers\FacebookRedirectLoginHelper->getAccessToken()
#6 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\FB->callback()
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FB))
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#11 {main}
CRITICAL - 2021-02-01 21:46:26 --> Resolving timed out after 10000 milliseconds
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\FacebookClient.php(216): Facebook\HttpClients\FacebookCurlHttpClient->send('https://graph.f...', 'GET', '', Array, 60)
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(277): Facebook\FacebookClient->sendRequest(Object(Facebook\FacebookRequest))
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(226): Facebook\Authentication\OAuth2Client->sendRequestWithClientParams('/oauth/access_t...', Array)
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(166): Facebook\Authentication\OAuth2Client->requestAnAccessToken(Array)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Helpers\FacebookRedirectLoginHelper.php(228): Facebook\Authentication\OAuth2Client->getAccessTokenFromCode('AQAuo--HlomuCqG...', 'http://localhos...')
#5 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\FB.php(41): Facebook\Helpers\FacebookRedirectLoginHelper->getAccessToken()
#6 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\FB->callback()
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FB))
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#11 {main}
CRITICAL - 2021-02-01 21:48:50 --> Resolving timed out after 10000 milliseconds
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\FacebookClient.php(216): Facebook\HttpClients\FacebookCurlHttpClient->send('https://graph.f...', 'GET', '', Array, 60)
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(277): Facebook\FacebookClient->sendRequest(Object(Facebook\FacebookRequest))
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(226): Facebook\Authentication\OAuth2Client->sendRequestWithClientParams('/oauth/access_t...', Array)
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(166): Facebook\Authentication\OAuth2Client->requestAnAccessToken(Array)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Helpers\FacebookRedirectLoginHelper.php(228): Facebook\Authentication\OAuth2Client->getAccessTokenFromCode('AQBcmOVJf8TalVJ...', 'http://localhos...')
#5 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\FB.php(41): Facebook\Helpers\FacebookRedirectLoginHelper->getAccessToken()
#6 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\FB->callback()
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FB))
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#11 {main}
CRITICAL - 2021-02-01 21:54:52 --> Resolving timed out after 10000 milliseconds
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\FacebookClient.php(216): Facebook\HttpClients\FacebookCurlHttpClient->send('https://graph.f...', 'GET', '', Array, 60)
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(277): Facebook\FacebookClient->sendRequest(Object(Facebook\FacebookRequest))
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(226): Facebook\Authentication\OAuth2Client->sendRequestWithClientParams('/oauth/access_t...', Array)
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(166): Facebook\Authentication\OAuth2Client->requestAnAccessToken(Array)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Helpers\FacebookRedirectLoginHelper.php(228): Facebook\Authentication\OAuth2Client->getAccessTokenFromCode('AQClgzpgNLYaORO...', 'http://localhos...')
#5 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\FB.php(41): Facebook\Helpers\FacebookRedirectLoginHelper->getAccessToken()
#6 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\FB->callback()
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FB))
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#11 {main}
CRITICAL - 2021-02-01 21:57:21 --> Resolving timed out after 10000 milliseconds
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\FacebookClient.php(216): Facebook\HttpClients\FacebookCurlHttpClient->send('https://graph.f...', 'GET', '', Array, 60)
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(277): Facebook\FacebookClient->sendRequest(Object(Facebook\FacebookRequest))
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(226): Facebook\Authentication\OAuth2Client->sendRequestWithClientParams('/oauth/access_t...', Array)
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(166): Facebook\Authentication\OAuth2Client->requestAnAccessToken(Array)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Helpers\FacebookRedirectLoginHelper.php(228): Facebook\Authentication\OAuth2Client->getAccessTokenFromCode('AQCftCfkNJP2gdU...', 'http://localhos...')
#5 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\FB.php(41): Facebook\Helpers\FacebookRedirectLoginHelper->getAccessToken()
#6 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\FB->callback()
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FB))
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#11 {main}
CRITICAL - 2021-02-01 21:58:01 --> Resolving timed out after 10000 milliseconds
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\FacebookClient.php(216): Facebook\HttpClients\FacebookCurlHttpClient->send('https://graph.f...', 'GET', '', Array, 60)
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(277): Facebook\FacebookClient->sendRequest(Object(Facebook\FacebookRequest))
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(226): Facebook\Authentication\OAuth2Client->sendRequestWithClientParams('/oauth/access_t...', Array)
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Authentication\OAuth2Client.php(166): Facebook\Authentication\OAuth2Client->requestAnAccessToken(Array)
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\facebook\graph-sdk\src\Facebook\Helpers\FacebookRedirectLoginHelper.php(228): Facebook\Authentication\OAuth2Client->getAccessTokenFromCode('AQCmDtt-873Y5kZ...', 'http://localhos...')
#5 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\FB.php(41): Facebook\Helpers\FacebookRedirectLoginHelper->getAccessToken()
#6 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\FB->callback()
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FB))
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#11 {main}
CRITICAL - 2021-02-01 22:08:52 --> syntax error, unexpected '}', expecting ';' or ','
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\Users\\X550JX...')
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(869): class_exists('\\App\\Controller...', true)
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(394): CodeIgniter\CodeIgniter->startController()
#6 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#9 {main}
CRITICAL - 2021-02-01 22:08:53 --> syntax error, unexpected '}', expecting ';' or ','
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Router\Router.php(195): CodeIgniter\Router\Router->autoRoute('FB/callback')
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(793): CodeIgniter\Router\Router->handle('FB/callback')
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(364): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#6 {main}
CRITICAL - 2021-02-01 22:09:02 --> syntax error, unexpected '}', expecting ';' or ','
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\Users\\X550JX...')
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(869): class_exists('\\App\\Controller...', true)
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(394): CodeIgniter\CodeIgniter->startController()
#6 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#9 {main}
CRITICAL - 2021-02-01 22:09:38 --> syntax error, unexpected '}', expecting ';' or ','
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\Users\\X550JX...')
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(869): class_exists('\\App\\Controller...', true)
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(394): CodeIgniter\CodeIgniter->startController()
#6 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#9 {main}
CRITICAL - 2021-02-01 22:25:52 --> Unable to connect to the database.
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `email`\n...', Array, false)
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(404): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Model.php(436): CodeIgniter\Model->find()
#4 C:\Users\X550JX\dev\PHP\cidatabase\app\Controllers\Home.php(60): CodeIgniter\Model->findColumn('email')
#5 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->login()
#6 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#10 {main}
CRITICAL - 2021-02-01 22:47:34 --> Too few arguments to function App\Controllers\FB::login(), 0 passed in C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\FB->login()
#1 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FB))
#2 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\Users\X550JX\dev\PHP\cidatabase\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 C:\Users\X550JX\dev\PHP\cidatabase\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\Users\\X550JX...')
#5 {main}
